content/en/docs/languages/js/automatic/configuration.md

```diff
@@ -39,3 +39,13 @@ For example, to only enable the `env` and `host` detectors, you can set:
 ```shell
 OTEL_NODE_RESOURCE_DETECTORS=env,host
 ```
+
+### Excluding auto-instrumentation
+
+By default, the Node.js auto-instrumentation has all the
+[supported instrumentation libraries](https://github.com/open-telemetry/opentelemetry-js-contrib/blob/main/metapackages/auto-instrumentations-node/README.md#supported-instrumentations)
+enabled.
+This makes instrumentation easy, but can result in slow application startup time and collecting too much or unwanted data.
+
+To enable only specific instrumentations you can use the `OTEL_NODE_ENABLED_INSTRUMENTATIONS` environment variable as documented in the
+[Node.js auto-instrumentation documentation](https://github.com/open-telemetry/opentelemetry-js-contrib/tree/main/metapackages/auto-instrumentations-node#usage-auto-instrumentation).
```

Comment:
Thanks for updating this, Also here, do not reference the documentation from the package, but bring the documentation to the website, as opentelemetry.io should be the place where documentation lives.

Reply:
> bring the documentation to the website

Regarding the `Node.js auto-instrumentation documentation` link: I overwrote this whole change based on the documentation from repo, so that link isn't needed any more.

Reply:
@svrnm,
> bring the documentation to the website

Regarding the `supported instrumentation libraries` link: I thought about inlining that as well.

My first preference is to keep it as it is, because:
* I just moved this link here from k8s-operator page
* It links to single source of truth for documentation (although in git repo, it is documentation, not code)
* [Supported libraries and frameworks](https://opentelemetry.io/docs/languages/js/automatic/#supported-libraries-and-frameworks) in the same repo also links to the same place (single source of truth)

Perhaps I should also link from here to [Supported libraries and frameworks](https://opentelemetry.io/docs/languages/js/automatic/#supported-libraries-and-frameworks) instead?

I'd prefer not to change existing correct information in this documentation project, although I wouldn't mind if someone else did it.

Reply:
> It links to single source of truth for documentation (although in git repo, it is documentation, not code)

The single source of truth should be docs and not the READMEs in the code repos, we have similar motions going on for other languages & their automatic/zero-code instrumentations, but we can resolve this via a separate PR, especially since (as you stated) we have that kind of reference at different places already and this is not the purpose of this specific PR 

@open-telemetry/javascript-maintainers , @open-telemetry/docs-maintainers please take a look

Reply:
> we can resolve this via a separate PR, especially since (as you stated) we have that kind of reference at different places already and this is not the purpose of this specific PR

@svrnm , thanks! Could you also approve this PR?